src/bopforge/continuous_pipeline/functions.py

```diff
@@ -412,10 +412,25 @@ def get_linear_model_summary(
     sign = np.sign(pred)
     if np.any(np.prod(inner_ui[:, index], axis=0) < 0):
         summary["score"] = float("nan")
+        summary["star_rating"] = 0
     else:
-        summary["score"] = float(
+        score = float(
             ((sign * burden_of_proof)[:, index].mean(axis=1)).min()
         )
+        summary["score"] = score
+        #Assign star rating based on ROS
+        if np.isnan(score):
+            summary["star_rating"] = 0
+        elif score > 0.6152:
```

Comment:
I think here instead of using the number, we should use
```
np.log(1 + 0.85)
```
to make it more interpretable :)

Reply:
I've updated the bounds – thanks for this suggestion!